src/iterators/windows.rs

```diff
@@ -20,6 +20,23 @@ impl<'a, A, D: Dimension> Windows<'a, A, D> {
         E: IntoDimension<Dim = D>,
     {
         let window = window_size.into_dimension();
+        let ndim = window.ndim();
+
+        let mut unit_stride = D::zeros(ndim);
+        let stride_slice = unit_stride.slice_mut();
```

Comment:
Since you asked, I think `unit_stride.slice_mut().fill(1);` would work here.

Reply:
Ahh was totally unaware this was possible, thank you!